src/simmi/While.java

```diff
@@ -0,0 +1,50 @@
+package simmi;
+
+public class While {
+	
+	
+	void even(int num1, int num2) {
+		int num = num1;
```

Comment:
you don’t require to create extra variable, you can handle this without num, think on this

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files